### ISSUE TITLE
关于data_loader模块中collate_fn函数添加@staticmethod

### ISSUE THREAD

**Issue by cherryMonth** [thunlp/OpenNRE#290]
def collate_fn(data):
        data = list(zip(*data))
        labels = data[0]
        seqs = data[1:]
        batch_labels = torch.tensor(labels).long()  # (B)
        batch_seqs = []
        for seq in seqs:
            batch_seqs.append(torch.cat(seq, 0))  # (B, L)
        return [batch_labels] + batch_seqs

![image](https://user-images.githubusercontent.com/28805875/93435967-bcba6280-f8fc-11ea-97a7-bbd72beccec0.png)

当前函数是类SentenceREDataset的一个实例函数但是没有使用实例的变量，添加staticmethod装饰器合适一些。
